tests/mock_network/churn.rs

```diff
@@ -165,20 +393,10 @@ fn add_nodes<R: Rng>(
             .create();
         if let Some(&pfx) = prefixes.iter().find(|pfx| pfx.matches(&node.name())) {
             assert!(prefixes.remove(&pfx));
-            if skip_some_prefixes && !rng.gen_weighted_bool(prefixes.len() as u32) {
```

Comment:
Wonder if this still make sense i.e not adding nodes to all prefixes? can be re-added in later PR.

Reply:
Hm, I think I removed this accidentally. I'll add it back in a followup PR, as you are suggesting.